README.md

```diff
@@ -234,6 +242,37 @@ all the attributes associated to this particular CBV.
 The list of all CBVs and the respective attributes can be viewed in ./src/cbv/cbv.js.
 
 
+### New constants
```

Comment:
This section has to be changed.

First, don't say 'new' in the doc, otherwise it will be considered as new in 10 years.

Second, you need to add explanations on how to use the constants, not how they are declared (What do I need to import? - How do I call it? - etc). You can remove the eventEpcRelType from the doc, it is a very specific example that almost no one will use I think.